### PR TITLE
Remove custom variables and margin

### PR DESCRIPTION
**Pull request by SMahil**
Custom variables have been removed for more usability. Instead of that bootstrap classes has been used

Issue: #404

### CODE REVIEW

File: src/app/board/board.component.scss [hunk 1]
```diff
@@ -6,20 +6,21 @@
   margin: em(20) auto;
   padding-top: em(96);
   display: flex;
-  background-color: rgba($color-pf-black-100, 0.2);
+  background-color: $color-pf-white;
```
Comment (sanbornsen) [inline src/app/board/board.component.scss]: I believe the previous one used to make the background transparent. Will it work the same way now?

Comment (SMahil) [inline src/app/board/board.component.scss]: yes it is

**Comment by joshuawilson** [comment]
@SMahil please rebase

**Comment by joshuawilson** [comment]
@SMahil what does this PR fix or do? What is the issue #?

**Pull request merged**